core/store/migrate/migrations/0232_add_workflow_spec.sql

```diff
@@ -0,0 +1,67 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE workflow_specs (
+    id              SERIAL PRIMARY KEY,
+    workflow_id     varchar(64) NOT NULL,
+    workflow        text NOT NULL,
+    created_at      timestamp with time zone NOT NULL,
+    updated_at      timestamp with time zone NOT NULL
+);
+
+INSERT INTO workflow_specs (workflow_id, workflow, created_at, updated_at)
+    VALUES ('15c631d295ef5e32deb99a10ee6804bc4af1385568f9b3363f6552ac6dbb2cef', '<DUMMY WORKFLOW>', NOW(), NOW());
```

Comment:
why this?

Reply:
This was needed to be able to deploy this PR to staging -- without it we fail the constraint. I was hesitant to remove this after deployment on the off chance that the migration will fail for others (although I don't really see how!)

Reply:
Having thought about this more, I'd like to keep this in out of an abundance of caution -- I've added the hardcoded workflow as the default and added some default values for the WorkflowOwner / Workflow ID.